flashinfer_trace/tests/references/test_moe_fp8_block_scale_ds_routing_topk8_ng1_kg1_e256_h3072_i1536.py

```
@@ -0,0 +1,236 @@
+import torch
+from flashinfer.fused_moe import trtllm_fp8_block_scale_moe
+
+# MiniMax M2: E=256, H=3072, I=1536, topk=8, DeepSeek routing (type 2), n_group=1, topk_group=1
+E_GLOBAL = 256
+H = 3072
+I = 1536
```

Ruff (0.15.9)

[error] Ambiguous variable name: `I`

(E741)